### PR TITLE
Translate all strings inside of 'Text' component

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

File: app/core/src/screens/homepage/Homepage.js [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 import * as React from 'react';
 import { View } from 'react-native';
-import { Button, Layout, Text } from '@kiwicom/react-native-app-shared';
+import { Button, Layout } from '@kiwicom/react-native-app-shared';
 import { type NavigationType } from '@kiwicom/react-native-app-navigation';
-import Translation from '@kiwicom/react-native-app-translations';
+import { DummyTranslation } from '@kiwicom/react-native-app-translations';
```
Comment (mrtnzlml) [inline app/core/src/screens/homepage/Homepage.js]: Now it's time to think about a better name for "DummyTranslation"... 😄

File: app/hotels/src/singleHotel/hotelInformation/description/Facilities.js [hunk 2]
```diff
@@ -11,8 +11,13 @@ import {
   Text,
   Touchable,
 } from '@kiwicom/react-native-app-shared';
+import Translations, {
+  DummyTranslation,
+  TranslationFragment,
+} from '@kiwicom/react-native-app-translations';
 
 import type { Facilities_facilities } from './__generated__/Facilities_facilities.graphql';
+import Translation from '../../../../../translations/src/Translation';
```
Comment (mrtnzlml) [inline app/hotels/src/singleHotel/hotelInformation/description/Facilities.js]: TODO: FIX!

File: app/core/src/screens/homepage/Homepage.js (hunk 1 above)
Comment (tbergq) [reply app/core/src/screens/homepage/Homepage.js]: Agreed!

Comment (mrtnzlml) [reply app/core/src/screens/homepage/Homepage.js]: Ideas?

Comment (tbergq) [reply app/core/src/screens/homepage/Homepage.js]: I will sleep on it 🛌

**Pull request merged**